Azure-Samples/azure-search-openai-demo
# [BugFix] Fix logic in prepdocs to always update search index

## Purpose

Fixes #1063
I think this line of code was accidentally indented.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Delete an md5 file, run scripts/prepdocs.sh, see logs from update_content call:

Splitting 'role_library.pdf' into sections
        Uploading blob for whole file -> role_library.pdf
Batch Completed. Batch size  16 Token count 3246
Batch Completed. Batch size  16 Token count 3167
Batch Completed. Batch size  16 Token count 3201
Batch Completed. Batch size  10 Token count 1949
